tests/integration/session/test_login.py

```diff
@@ -11,6 +11,8 @@
 
 SCHEMA_PATH_MAP = get_schema_path_map(include_test_schemas=True)
 
+# pylint: disable=too-many-public-methods
```

Comment:
Maybe just make some of the ones (that are being called in tests) private?

Reply:
Even renaming single method, `cir_url_mock`  as `_cir_url_mock` will cause this pylint issue go away? But you could rename all the static methods.

Reply:
Not sure this would work as I tried this and it failed both unit tests and linting

Reply:
Maybe you refactored it incorrectly? I renamed it to `_cir_url_mock` and both pylint (after removing too many public methods exclusion) and unit tests are happy. I am on your branch, I've made sure I re-installed all dependencies etc. What does it fail on for you?
EDIT: In your failed test a method cannot be found:
`AttributeError: 'TestLoginWithPostRequest' object has no attribute 'schema_url_mock'` so some bad refactoring.

Reply:
There was an `_` missing somewhere else too, updated 👍

Reply:
Can you rename all remaining static methods (3 left?) so we have some consistency in the file?

Reply:
Renamed them 👍